beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -651,12 +639,6 @@ pub static DEFAULT_ETH1_VOTES: LazyLock<Result<IntCounter>> = LazyLock::new(|| {
 /*
  * Chain Head
  */
-pub static UPDATE_HEAD_TIMES: LazyLock<Result<Histogram>> = LazyLock::new(|| {
-    try_create_histogram(
-        "beacon_update_head_seconds",
-        "Time taken to update the canonical head",
-    )
-});
```

Comment:
We have `FORK_CHOICE_TIMES` which cover what we may be interested in, ok to remove